.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
These secrets are named the same ;

```
   TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
   TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
```

Reply:
Actually make sense, I didn't notice just copied and pasted 😁 let me change